Bricks/cerealize/cerealize.h

```diff
@@ -212,7 +204,7 @@ class CerealBinaryFileParser {
     if (fi_.peek() != std::char_traits<char>::eof()) {  // This is safe with any next byte in file.
       std::unique_ptr<T_ENTRY> entry;
       si_(entry);
-      processor(*entry.get());
+      processor(std::move(*entry.get()));
```

Comment:
It should -- the pointer is not `const`. It just makes sure if `processor()` accepts an `E&&`, then a move constructor is used (and the contents of the object stored inside the `unique_ptr<>` here is invalidated.)